aten/src/ATen/native/native_functions.yaml

```diff
@@ -1197,6 +1197,10 @@
     CPU: floor_out
     CUDA: floor_out
 
+- func: floor_divide(Tensor input, Tensor other) -> Tensor
+
+- func: floor_divide.Scalar(Tensor input, Scalar other) -> Tensor
```

Comment:
named tensor support?

Reply:
`supports_named_tensor` defaults to true (which is intended here). I don't know if it's convention to manually specify it on the declaration. cc @zou3519

Reply:
Thanks for flagging, @xuhdev .

@eellison `supports_named_tensor` defaults to `False`. This function looks simple enough that it should support named tensors; could you add `supports_named_tensor: True` and a simple test in https://github.com/pytorch/pytorch/blob/master/test/test_namedtensor.py ?

Reply:
added i think!

Reply:
Yeah, it looks great. Thanks for adding it!